src/endpoints/listPublicAPIKeys.ts

```diff
@@ -0,0 +1,53 @@
+import { requestUserApi } from '../requestApi.js';
+import { LocalConfiguration } from '../types.js';
+
+interface ListPublicAPIKeys {
+    localConfiguration: LocalConfiguration;
+}
+
+export interface ListPublicAPIKeysOutput {
+    /**
+     * List of api keys
+     */
+    publicAPIKeys: {
+        /**
+         * The creation date
+         */
+        creationDateUnix: number;
+        /**
+         * The last modification date
+         */
+        updateDateUnix: number;
+        /**
+         * The last invalidation date
+         */
+        invalidationDateUnix: null | number;
+        /**
+         * The access key
+         */
+        accessKey: string;
+        /**
+         * The description of the api key
+         */
+        description: string;
+        /**
+         * Platform used to create the report api key
```

Comment:
isnt't it just public api same just after ?

Reply:
yeah forgot to rename